lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -34,14 +35,24 @@ class PartialFinder < TemplateFinder
         # @since 0.4.3
         # @api private
         def find
-          if path = partial_template_under_view_path
-            View::Template.new(path, @view.configuration.default_encoding)
-          else
-            super
-          end
+          find_cached_template
         end
 
         protected
+
+        # @since x.x.x
+        # @api private
+        def find_cached_template
+          conf = Lotus::View::Configuration.for(@view)
```

Comment:
@stevehook `@view` already has its own configuration associated: `@view.configuration`. Doing a lookup with `Configuration.for` is more expensive.